app/screens/home/RouteList.tsx

```diff
@@ -444,11 +504,15 @@ const RouteList = ({
             <DeFiButton mode="contained" onPress={() => setIsStakingRouteEdit(true)} style={{ flex: 1 }}>
               {t("model.route.staking")}
             </DeFiButton>
+            <SpacerH />
+            <DeFiButton mode="contained" onPress={() => setIsCryptoRouteEdit(true)} style={{ flex: 1 }}>
+              {t("model.route.crypto")}
+            </DeFiButton>
```

Comment:
Visible for non-beta user
![image](https://user-images.githubusercontent.com/85513542/179792595-93c46f05-2361-47ba-9fbc-60e1145dede6.png)